nengo/tests/test_learning_rules.py

```diff
@@ -70,39 +72,45 @@ def _test_pes(Simulator, nl, plt, seed,
     assert not np.allclose(weights[0], weights[-1], atol=1e-5)
 
 
-def test_pes_ens_ens(Simulator, nl_nodirect, plt, seed):
+@pytest.mark.parametrize('apply_every', [None, 1e-5, 0.005])
```

Comment:
`0` should behave the same as `1e-5`, any value of `apply_every < dt` should be identical. I have tested `0` also and it's fine; any argument to use a test value other than `1e-5`... it is somewhat arbitrary and I have no attachment to it? (negative numbers should also behave the same but they are not valid and raise an exception)